packages/wallet/src/communication/actions.ts

```diff
@@ -132,7 +132,7 @@ export function isCommonAction(
     (action.type === 'WALLET.COMMON.COMMITMENTS_RECEIVED' ||
       action.type === 'WALLET.COMMON.COMMITMENT_RECEIVED') &&
     // When passed a protocol, check that it's got the protocol in the protocol locator
-    (!protocol || action.protocolLocator.indexOf(protocol) >= 0)
+    (!protocol || (action.protocolLocator && action.protocolLocator.indexOf(protocol) >= 0))
```

Comment:
Why is this needed? Both `COMMITMENTS_RECEIVED` and `COMMITMENT_RECEIVED` have a `protocolLocator`

Reply:
I should investigate this further, but `isCommonAction` was receiving a `COMMITMENT_RECEIVED` with an `undefined` protocolLocator from a concluding protocol.

Reply:
I thought `COMMITMENT_RECEIVED` now has a mandatory `protocolLocator` property?

Reply:
This looks like it's due to firebase deciding to strip `protocolLocator:[]` from the action. Going to leave this is as now and we can investigate further alter.